GTG/gtk/browser/cell_renderer_tags.py

```diff
@@ -16,17 +16,20 @@
 # this program.  If not, see <http://www.gnu.org/licenses/>.
 # -----------------------------------------------------------------------------
 
+from gi.repository import GObject, GLib, Gtk, Gdk
+from gi.repository import Pango
 import gi
 import cairo
-gi.require_version('PangoCairo', '1.0')
-
-from gi.repository import Pango
-from gi.repository import PangoCairo
-from gi.repository import GObject, GLib, Gtk, Gdk
 
 
 class CellRendererTags(Gtk.CellRenderer):
 
+    # XXX: these 2 lines were moved within the class to keep the PEP8 checker
+    #      from reporting an E402 error due to require_version needing to be
+    #      called before the PangoCairo import
+    gi.require_version('PangoCairo', '1.0')
+    from gi.repository import PangoCairo
+
```

Comment:
Couldn't we use something like `# nopep8` to bypass the error?

Reply:
Ah thank you - I was looking for that feature in the pycodestyle docs but didn't see it

Reply:
I've seen it in some PR while looking. I looked at the [pycodestyle documentation](https://pycodestyle.pycqa.org/en/latest/intro.html#related-tools), and it suggests `# noqa` instead.

Reply:
Ah...I thought `# noqa` was reserved for the error codes marked with the `(^)`, but it works for this - updating it